Packages/com.unity.inputsystem/InputSystem/Editor/UITKAssetEditor/Views/CopyPasteHelper.cs

```diff
@@ -385,7 +372,7 @@ public static SerializedProperty DuplicateElement(SerializedProperty arrayProper
 
         #endregion
 
-        private static List<SerializedProperty> GetBindingsForComposite(SerializedProperty bindingsArray, int indexOfComposite)
+        public static List<SerializedProperty> GetBindingsForComposite(SerializedProperty bindingsArray, int indexOfComposite)
```

Comment:
Possibly internal rather than public?

Reply:
fixed in [32a3a64](https://github.com/Unity-Technologies/InputSystem/pull/1831/commits/32a3a649dfd5f67ddb511a9aa4659e88de27b84d)